test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/CaseSensitiveTagHelperAttributeComparer.cs

```diff
@@ -22,16 +21,13 @@ public bool Equals(IReadOnlyTagHelperAttribute attributeX, IReadOnlyTagHelperAtt
                 attributeX == attributeY ||
                 // Normal comparer doesn't care about the Name case, in tests we do.
                 string.Equals(attributeX.Name, attributeY.Name, StringComparison.Ordinal) &&
-                Equals(attributeX.Value, attributeY.Value);
+                Equals(attributeX.Value, attributeY.Value) &&
+                attributeX.Minimized == attributeY.Minimized;
```

Comment:
Rebase and `Minimized` will be handled correctly in the base class.

Unless you want to check both `Minimized` and `Value` even when `Minimzed == true`.  In that case, add a bit more to the comment at line 22.